extensions/ql-vscode/src/codeql-cli/distribution.ts

```diff
@@ -475,19 +494,41 @@ class ExtensionSpecificDistributionManager {
   }
 
   private createReleasesApiConsumer(): ReleasesApiConsumer {
-    const ownerName = this.config.ownerName
-      ? this.config.ownerName
-      : DEFAULT_DISTRIBUTION_OWNER_NAME;
-    const repositoryName = this.config.repositoryName
-      ? this.config.repositoryName
-      : DEFAULT_DISTRIBUTION_REPOSITORY_NAME;
     return new ReleasesApiConsumer(
-      ownerName,
-      repositoryName,
+      this.distributionOwnerName(),
+      this.distributionRepositoryName(),
       this.config.personalAccessToken,
     );
   }
 
+  private distributionOwnerName(): string {
+    if (this.config.ownerName) {
+      return this.config.ownerName;
+    } else if (this.config.channel === "nightly") {
+      return NIGHTLY_DISTRIBUTION_OWNER_NAME;
+    } else {
+      return DEFAULT_DISTRIBUTION_OWNER_NAME;
+    }
+  }
+
+  private distributionRepositoryName(): string {
+    if (this.config.repositoryName) {
+      return this.config.repositoryName;
+    } else if (this.config.channel === "nightly") {
+      return NIGHTLY_DISTRIBUTION_REPOSITORY_NAME;
+    } else {
+      return DEFAULT_DISTRIBUTION_REPOSITORY_NAME;
+    }
+  }
+
+  private usingNightlyReleases(): boolean {
+    return (
+      !this.config.ownerName &&
```

Comment:
I find it odd that you can set a custom owner name and repo name separately, and it'll maybe use a custom org name with a default value for repo name. I'm not sure why someone would want that, but that's what we have. Therefore we have to check if either is set, and we can't just combine our values together into one NWO.